src/common/components/modal-dialog/modal-dialog.const.ts

```diff
@@ -1,4 +1,5 @@
 export const CANVAS_SETTINGS_TITLE = 'Canvas Settings';
+export const ADD_RELATION_TITLE = 'Add relation';
 export const EDIT_RELATION_TITLE = 'Edit relation';
 export const ADD_COLLECTION_TITLE = 'Add Collection';
```

Comment:
All the code is correct. However, you can change the value to uppercase 'R' in the word 'relationship' in both constants (ADD_RELATION_TITLE, EDIT_RELATION_TITLE), for consistency . Thank you.